src/components/org/OrgChart/OrgChart.stories.tsx

```diff
@@ -231,7 +250,7 @@ const OrgChartStory = () => {
     const componentDisplayType = useComponentDisplayType();
     const cardHeight = componentDisplayType === 'Compact' ? 110 : 142;
     const rowMargin = componentDisplayType === 'Compact' ? 120 : 164;
-    const cardMargin = componentDisplayType === 'Compact' ? 24 : 32;
+    const cardMargin = componentDisplayType === 'Compact' ? 24 : 16;
```

Comment:
Could we leave this as 32? It's whats being used in the pro-org app

Reply:
Changed back to 32